src/node.rs

```diff
@@ -130,6 +130,29 @@ impl<T> Node<T> {
     }
 
     ///
+    /// Exchange the attached data through a new one
+    ///
+    /// Returns the old value of data
+    ///
+    /// ```
+    /// use id_tree::Node;
+    ///
+    /// let mut node_four: Node<i32> = Node::new(3);
+    ///
+    /// // ops! lets correct this
+    /// let three = node_four.replace_data(4);
+    ///
+    /// assert_eq!(node_four.data(), &4);
+    /// assert_eq!(three, 3);
+    /// ```
+    ///
+    pub fn replace_data(&mut self, mut data: T) -> T
+    {
```

Comment:
Sure thing,
you might be interested in adding a [rustfmt.toml](https://github.com/rust-lang-nursery/rustfmt) to your repo, because this might have happened more then one time to me and contributors could follow your guidelines more easily :)